pyiode/objects/table.pyx

```diff
@@ -1232,12 +1253,18 @@ cdef class Table:
         index: int
             index where to insert a line.
         value: int or str or list(str) or tuple(str) or TableLine
+            # GDM> the \n in the following lines seem odd. Do they actually do anything in a docstring?
             value of the new line.\n
-            If int, 'value' represents the type of the new line: LINE_TYPE_FILES, LINE_TYPE_MODE, 
+            # GDM> I *wonder* if an explicit and optional line_type argument would not be cleaner for the 'int' use case.
+            If int, 'value' represents the type of the new line: LINE_TYPE_FILES, LINE_TYPE_MODE,
+            # GDM> LINE_TYPE_SEPARATOR instead of LINE_TYPE_LINE would be clearer IMO
             LINE_TYPE_DATE, LINE_TYPE_LINE.\n
             If str, 'value' represents either a separator line if it only contains characters '-' 
             or a title line.
             If an iterable of str, 'value' represents the content of the cells of the new line.\n
+        # GDM> I suppose this mimics iode internal behavior and/or gui and/or reports, but having after default to True is
```

Comment:
Yes this mimics the GUI behavior. I will simply remove the option *after* from the Python Table.insert() method.
Thanks for noticing it.

Reply:
See PR #584